code/modules/mob/living/carbon/carbon.dm

```diff
@@ -1295,6 +1295,14 @@
 /// Special carbon interaction on lying down, to transform its sprite by a rotation.
 /mob/living/carbon/proc/lying_angle_on_lying_down(new_lying_angle)
 	if(!new_lying_angle)
+		//SKYRAT EDIT ADDITION BEGIN - SOUNDS
+		if(dir == WEST)
+			set_lying_angle(270)
+			return
+		else if(dir == EAST)
+			set_lying_angle(90)
+			return
+		//SKYRAT EDIT END
 		set_lying_angle(pick(90, 270))
```

Comment:
This always picks 90, right... it's gotta be a list. well guess thats an upstream issue

Reply:
Pick can be a list or not a list

Reply:
interesting, I remember some issues from picks of non-lists